tree/ntuple/v7/inc/ROOT/RColumnElement.hxx

```diff
@@ -683,6 +698,74 @@ public:
    }
 };
 
+namespace BitPacking {
+
+using Word_t = std::uintmax_t;
+constexpr std::size_t kBitsPerWord = sizeof(Word_t) * 8;
+
+/// Returns the minimum safe size (in bytes) of a buffer that is intended to be used as a destination for PackBits
+/// or a source for UnpackBits.
+/// Passing a buffer that's less than this size will cause invalid memory reads and writes.
+constexpr std::size_t MinBufSize(std::size_t count, std::size_t nDstBits)
```

Comment:
It's supposed to also be used by an "external user" of the Pack/UnpackBits functions. In our case we don't need it though, because this happens to be the same as `GetBitsOnStorage()` (well, it's no coincidence of course - it was made this way intentionally). I can probably remove it and just say directly in the functions' comments that the proper size is `(count * nBits + 7) / 8`.